### PR TITLE
[Resolves #17] Fix docker image build

### LINKED ISSUE #17: docker image fails to build
Docker image fails to build due to a dependency change (packaging 22.0 was release on 12/7/2022).  We need to freshen up the docker image and make it buildable again.

```
➜  docker build .
[+] Building 39.9s (7/10)
 => [internal] load build definition from Dockerfile                                                                                                                                                                                     0.0s
 => => transferring dockerfile: 1.29kB                                                                                                                                                                                                   0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                                                                                                          0.0s
 => [internal] load metadata for docker.io/library/alpine:3.16                                                                                                                                                                           1.6s
 => [auth] library/alpine:pull token for registry-1.docker.io                                                                                                                                                                            0.0s
 => [1/6] FROM docker.io/library/alpine:3.16@sha256:b95359c2505145f16c6aa384f9cc74eeff78eb36d308ca4fd902eeeb0a0b161b                                                                                                                     0.8s
 => => resolve docker.io/library/alpine:3.16@sha256:b95359c2505145f16c6aa384f9cc74eeff78eb36d308ca4fd902eeeb0a0b161b                                                                                                                     0.0s
 => => sha256:b95359c2505145f16c6aa384f9cc74eeff78eb36d308ca4fd902eeeb0a0b161b 1.64kB / 1.64kB                                                                                                                                           0.0s
 => => sha256:3d426b0bfc361d6e8303f51459f17782b219dece42a1c7fe463b6014b189c86d 528B / 528B                                                                                                                                               0.0s
 => => sha256:bfe296a525011f7eb76075d688c681ca4feaad5afe3b142b36e30f1a171dc99a 1.47kB / 1.47kB                                                                                                                                           0.0s
 => => sha256:ca7dd9ec2225f2385955c43b2379305acd51543c28cf1d4e94522b3d94cce3ce 2.81MB / 2.81MB                                                                                                                                           0.5s
 => => extracting sha256:ca7dd9ec2225f2385955c43b2379305acd51543c28cf1d4e94522b3d94cce3ce                                                                                                                                                0.2s
 => [2/6] RUN apk add --update --no-cache     patch     git     openssh     ca-certificates     build-base     curl     bzip2-dev     bash     py3-pip     python3     python3-dev     libffi-dev     ruby     ruby-dev     ruby-bundl  31.8s
 => ERROR [3/6] RUN pip3 install --no-cache-dir virtualenv     && pip3 install --no-cache-dir tox     && addgroup -g 3434 circleci     && adduser -D -u 3434 -G circleci -s /bin/bash circleci                                           5.7s
------
 > [3/6] RUN pip3 install --no-cache-dir virtualenv     && pip3 install --no-cache-dir tox     && addgroup -g 3434 circleci     && adduser -D -u 3434 -G circleci -s /bin/bash circleci:
#7 2.328 Collecting virtualenv
#7 2.383   Downloading virtualenv-20.17.1-py3-none-any.whl (8.8 MB)
#7 2.958      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 8.8/8.8 MB 15.5 MB/s eta 0:00:00
#7 3.062 Collecting filelock<4,>=3.4.1
#7 3.070   Downloading filelock-3.9.0-py3-none-any.whl (9.7 kB)
#7 3.107 Collecting platformdirs<3,>=2.4
#7 3.115   Downloading platformdirs-2.6.2-py3-none-any.whl (14 kB)
#7 3.139 Collecting distlib<1,>=0.3.6
#7 3.149   Downloading distlib-0.3.6-py2.py3-none-any.whl (468 kB)
#7 3.165      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.5/468.5 kB 42.1 MB/s eta 0:00:00
#7 3.283 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#7 3.683 Successfully installed distlib-0.3.6 filelock-3.9.0 platformdirs-2.6.2 virtualenv-20.17.1
#7 3.684 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#7 4.747 Collecting tox
#7 4.788   Downloading tox-4.1.0-py3-none-any.whl (144 kB)
#7 4.800      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 144.3/144.3 kB 22.1 MB/s eta 0:00:00
#7 4.898 Collecting packaging>=22
#7 4.906   Downloading packaging-22.0-py3-none-any.whl (42 kB)
#7 4.912      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 42.6/42.6 kB 55.0 MB/s eta 0:00:00
#7 4.918 Requirement already satisfied: tomli>=2.0.1 in /usr/lib/python3.10/site-packages (from tox) (2.0.1)
#7 4.919 Requirement already satisfied: filelock>=3.8.2 in /usr/lib/python3.10/site-packages (from tox) (3.9.0)
#7 4.963 Collecting colorama>=0.4.6
#7 4.971   Downloading colorama-0.4.6-py2.py3-none-any.whl (25 kB)
#7 4.976 Requirement already satisfied: virtualenv>=20.17.1 in /usr/lib/python3.10/site-packages (from tox) (20.17.1)
#7 4.995 Collecting chardet>=5.1
#7 5.006   Downloading chardet-5.1.0-py3-none-any.whl (199 kB)
#7 5.016      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 199.1/199.1 kB 49.1 MB/s eta 0:00:00
#7 5.039 Collecting pyproject-api>=1.2.1
#7 5.047   Downloading pyproject_api-1.2.1-py3-none-any.whl (11 kB)
#7 5.087 Collecting cachetools>=5.2
#7 5.095   Downloading cachetools-5.2.0-py3-none-any.whl (9.3 kB)
#7 5.099 Requirement already satisfied: platformdirs>=2.6 in /usr/lib/python3.10/site-packages (from tox) (2.6.2)
#7 5.130 Collecting pluggy>=1
#7 5.138   Downloading pluggy-1.0.0-py2.py3-none-any.whl (13 kB)
#7 5.252 Requirement already satisfied: distlib<1,>=0.3.6 in /usr/lib/python3.10/site-packages (from virtualenv>=20.17.1->tox) (0.3.6)
#7 5.349 Installing collected packages: pluggy, packaging, colorama, chardet, cachetools, pyproject-api, tox
#7 5.371   Attempting uninstall: packaging
#7 5.373     Found existing installation: packaging 21.3
#7 5.380 ERROR: Cannot uninstall 'packaging'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
------
executor failed running [/bin/sh -c pip3 install --no-cache-dir virtualenv     && pip3 install --no-cache-dir tox     && addgroup -g 3434 circleci     && adduser -D -u 3434 -G circleci -s /bin/bash circleci]: exit code: 1
➜ 
```

### PR DESCRIPTION
**Pull request by zaro0508**
Base the sceptre docker image on the official python circleci docker image[1]
which supports pyenv, pipenv, poetry and node

[1] https://hub.docker.com/r/cimg/python

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,55 +1,9 @@
-FROM alpine:3.16
+FROM cimg/python:3.11.1-node
```
Comment (jfalkenstein) [inline Dockerfile:1]: I don't think we currently support 3.11, do we?

File: Dockerfile [hunk 2]
```diff
@@ -1,55 +1,9 @@
...
-
-ENV LANG=C.UTF-8 \
-    HOME=/home/circleci \
-    PATH=$PYENV_HOME/shims:$PYENV_HOME/bin:/home/circleci/.poetry/bin:$PATH
-
-RUN git clone --depth 1 https://github.com/pyenv/pyenv.git $PYENV_HOME \
-    && git clone https://github.com/pyenv/pyenv-virtualenv.git $(pyenv root)/plugins/pyenv-virtualenv \
-    && rm -rfv $PYENV_HOME/.git \
-    && pyenv install 3.7.10 \
+RUN pyenv install 3.7.10 \
```
Comment (jfalkenstein) [inline Dockerfile:3]: Is pyenv already installed on that base image?

File: Dockerfile (hunk 1 above)
Comment (zaro0508) [reply Dockerfile:1]: nope, but this is the latest image from circleci so might as well update to the lastest.  we don't need to test sceptre with python 3.11

File: Dockerfile (hunk 2 above)
Comment (zaro0508) [reply Dockerfile:3]: yes, pyenv is included in the base image.

**Pull request merged**